torch_geometric/nn/models/tgn.py

```diff
@@ -98,10 +99,10 @@ def forward(self, n_id: Tensor) -> Tuple[Tensor, Tensor]:
 
         return memory, last_update
 
-    def update_state(self, src: Tensor, dst: Tensor, t: Tensor,
-                     raw_msg: Tensor):
+    def update_state(self, batch: Batch):
         """Updates the memory with newly encountered interactions
         :obj:`(src, dst, t, raw_msg)`."""
+        src, dst, t, raw_msg = batch.src, batch.dst, batch.t, batch.msg
```

Comment:
```suggestion
    def update_state(self, src: Tensor, dst: Tensor, t: Tensor,
                     raw_msg: Tensor):
        """Updates the memory with newly encountered interactions
        :obj:`(src, dst, t, raw_msg)`."""
```
I would keep this as is. Because it doesn't break things, most models and model functions in pyg have this kind of signature, and this is explicit.